ui/components/HelmRepositoryDetail.tsx

```diff
@@ -45,6 +44,18 @@ function HelmRepositoryDetail({
   );
 }
 
+function tryLink(url: string) {
```

Comment:
Just a suggestion, maybe we could start adding utility/helper function return types (if not tests). Smth. like `React.ReactElement<any, any> | string` could work.